apps/mobile/src/libs/components/text/text.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactNode } from "react";
+import { TextStyle } from "react-native";
+
+import { RNText } from "~/libs/components/components";
```

Comment:
We don't need to export RNText from ~/libs/components/components. We will only use custom Text component in the app

Reply:
Done